go/test/endtoend/tabletmanager/throttler/throttler_test.go

```diff
@@ -73,7 +73,7 @@ var (
 )
 
 const (
-	throttlerInitWait            = 10 * time.Second
+	throttlerInitWait            = 15 * time.Second
```

Comment:
Any reason not to make it longer? If the init fails, a test still runs for up to 10-20 minutes (1 try, with potentially 1 retry, and the second retry doesn't seem to cause the CI workflow to pass); only then do you know it's failed and all we can do is re-run the workflow and wait again. Another 15 seconds at the beginning of the test feels much better. 🙂 I'd use 30 seconds here and increase the others below as well to 5, 30, 15, 15.

I'll approve so that you're unblocked as this is subjective — and what's here now may be all that's needed. 

Thank you!

Reply:
@mattlord you're right! I've increased this and a couple more timeouts even more.